modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/cards/NavigationCard.js

```diff
@@ -47,9 +47,11 @@ export default function NavigationCard({
 			title={title}
 			{...otherProps}
 		>
-			{!horizontal && symbol && <ClayIcon symbol={symbol} />}
-
-			{!horizontal && imageSrc && <img alt={imageAlt} src={imageSrc} />}
+			{!horizontal && imageSrc ? (
```

Comment:
Since this logic about rendering the children only when it's not horizontal is already defined in the component I guess we can simplify this to:

```
{imageSrc ? ( 
      <img alt={imageAlt} src={imageSrc} />
) : (
      symbol && <ClayIcon symbol={symbol} />
)}
```